arboretum/ibm_cloud/README.md

```diff
@@ -62,6 +62,49 @@ to include the fetchers and checks from this library in your downstream project.
    from arboretum.ibm_cloud.fetchers.fetch_cluster_list import ClusterListFetcher
    ```
 
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-ibm-cloud-cluster-resource]
+* Purpose: Write the resources of IBM Cloud Kubernetes clusters to the evidence locker.
+* Behavior: Retrieve the resources of IBM Cloud Kubernetes clusters listed by [cluster list fetcher][fetch-cluster-list]. For IBM Cloud Kubernetes Service (IKS) clusters, the fetcher [downloads cluster config via IBM Cloud API][ibm-cloud-download-config] and extract the access token from the config. For IBM Cloud Red Hat Kubernetes Service (ROKS) clusters, the fetcher retrieves the access token using [the OAUTH server][ibm-cloud-roks-oauth] of the cluster. TTL is set to 1 day.
+
+* Configuration elements:
+  * `org.ibm_cloud.accounts`
+    * Required
+    * List of accounts (string)
+    * Each account is an arbitrary name describing the IBM Cloud account. It is used to match to the token provided in the
+      credentials file in order for the fetcher to retrieve content from IBM Cloud for that account.
+  * `org.ibm_cloud.cluster_resources.target_resource_types`
+    * Optional
+    * List of resource types as strings
+    * See the document of the [Kubernetes resource fetcher][fetch-kube-cluster-resource] for details.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "ibm_cloud": {
+        "accounts": [
+          "myaccount1", "myaccount2"
+        ],
+        "cluster_resources": {
+          "target_resource_types": [
+            "secrets", "batch/v1/cronjobs", "apigroup.example.com/v1/mycustom"
+          ]
+        }
+      }
+    }
+  }
+  ```
+
+* Required credentials:
+  * `ibm_cloud` credentials with read/view permissions are needed for this fetcher to successfully retrieve the evidence.  See `Required credentials` of the [cluster list fetcher][fetch-cluster-list].
+* Import statement:
+
+   ```python
+   from arboretum.ibm_cloud.fetchers.fetch_cluster_resource import ClusterResourceFetcher
```

Comment:
`ClusterResourceFetcher` is also the name of the `kubernetes` version of this fetcher.  If added to the same import file this will cause a name collision and it won't be apparent why one of the fetchers (the first one) is not running.  Since the kubernetes version is already in place and we cannot really change the name of the class there, it would be a good idea to rename this one.  Maybe something like `ICClusterResourceFetcher`?

Reply:
I renamed it to as you suggested (`ICClusterResourceFetcher`).